cristin-biobank/src/main/java/no/unit/nva/biobank/fetch/FetchBiobankHandler.java

```diff
@@ -0,0 +1,46 @@
+package no.unit.nva.biobank.fetch;
+
+import static no.unit.nva.biobank.client.CristinBiobankApiClient.defaultClient;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.HttpURLConnection;
+import no.unit.nva.biobank.client.CristinBiobankApiClient;
+import no.unit.nva.biobank.model.QueryBiobank;
+import no.unit.nva.biobank.model.nva.Biobank;
+import no.unit.nva.cristin.common.handler.CristinHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+
+public class FetchBiobankHandler extends CristinHandler<Void, Biobank> {
```

Comment:
im thinking about removing CristinHandler because it mostly deals with language, and we now add that automatically. But it can be refactored when that happens